lib/LaTeXML/Package/pgfsys-latexml.def.ltxml

```diff
@@ -1031,7 +1095,6 @@ DefConstructor('\lxSVG@halign BoxSpecification',
     my $template = LaTeXML::Core::Alignment::Template->new((@nonreps ?
           (columns => [@nonreps], repeated => [@cols])
         : (columns => [@cols])));
-    #  print STDERR "Template = ".Stringify(Tokens(@template))."\n => ".$template->show."\n";
     # Now read & digest the body.
     # Note that the body MUST end with a \cr, and that we've made Special Arrangments
     # with \alignment@cr to recognize the end of the \halign
```

Comment:
The `pgfsys-latexml.def.ltxml` binding file has gotten a lot simpler and cleaner, very very nice.

Reply:
There was a lot of "Two wrongs makes a right" going on there. Which is challenging to debug, if you "fix" an "obvious" mistake and it gets worse!